exonum-node/Cargo.toml

```diff
@@ -0,0 +1,64 @@
+[package]
+name = "exonum-node"
+version = "0.13.0-rc.2"
+edition = "2018"
+authors = ["The Exonum Team <contact@exonum.com>"]
+homepage = "https://exonum.com/"
+repository = "https://github.com/exonum/exonum"
+documentation = "https://docs.rs/exonum"
```

Comment:
Nit: I think the link should be different. Not sure if we need to specify it now though.